index.js

```diff
@@ -19,10 +20,14 @@ if (args.length > 0) {
     state = null;
   }
   countryCode = COUNTRIES[passedCountry];
+  if (countryCode === "/au") {
+    skuList = SKUS_AUSTRALIA;
+    favorites = ["MMQX3X/A","MKH53X/A","MMQW3X","MK233X"];
```

Comment:
this is a great idea, to swap out the US favorite SKUs with the corresponding AU ones, but it might confuse people if they customize their SKUs in the `favorites` declaration, only to have them swapped out here. not sure what the best way to handle this would be - perhaps some sort of US -> AU SKU mapping? or perhaps we just leave it alone and leave it up to users to get their favorite SKUs correct?

Reply:
I've relocated it to the top of the file (line 10) and added a comment - would this reduce confusion?

I like the idea of a SKU mapping, but they may not always have a 1 to 1 mapping (plus it is one more thing to maintain).

Reply:
yeah, i think that'll have to do for now - not ideal, but prob the best we can do.